src/Web/Masa.Auth.Web.Admin.Rcl/Shared/AdminCompontentBase.cs

```diff
@@ -81,11 +84,16 @@ public string T(string key)
         else return I18n.T(key, true);
     }
 
-    public string T(string formatkey,params string[] args)
+    public string T(string formatkey, params string[] args)
     {
         return string.Format(T(formatkey), args);
     }
 
+    protected string DT(string key)
```

Comment:
What's the difference between DT and T?

Reply:
T get value from i18n file,DT get value from Dcc